sdcm/cluster.py

```diff
@@ -377,6 +377,22 @@ def host_id(self):
                 return full_nodetool_status[data_center][self.ip_address]['host_id']
         raise AssertionError(f"Could not find the requested node {self.ip_address} in nodetool status")
 
+    @property
+    def node_instance_type(self) -> Optional[str]:
```

Comment:
It should be `db_node_instance_type`,  isn't it?

Reply:
Correct - fixed

Reply:
Looks like you missed this change.